app/src/main/java/ru/startandroid/develop/sprint8v3/library/ui/fragment/PlaylistViewFragment.kt

```diff
@@ -0,0 +1,361 @@
+            override fun onSlide(bottomSheet: View, slideOffset: Float) {
+                binding.overlay.alpha = (slideOffset + 1f) / 2f
+            }
+        })
+
+        binding.overlay.setOnClickListener {
+            bottomMenuBehavior.state = BottomSheetBehavior.STATE_HIDDEN
+            bottomSheetBehavior.state = BottomSheetBehavior.STATE_COLLAPSED
+        }
+
+        binding.threePointsBtn.post {
+            val location = IntArray(2)
+            binding.threePointsBtn.getLocationOnScreen(location)
+            bottomSheetBehavior.peekHeight =
+                binding.root.height - location[1] - binding.threePointsBtn.height
+            binding.playlistName.getLocationOnScreen(location)
+            bottomMenuBehavior.peekHeight =
+                binding.root.height - location[1] - binding.playlistName.height
+        }
+
+        binding.threePointsBtn.setOnClickListener {
+            bottomMenuBehavior.state = BottomSheetBehavior.STATE_COLLAPSED
+        }
+        binding.shareBtn.setOnClickListener {
+            sharePlaylist()
+        }
+
+        binding.deletePlaylist.setOnClickListener {
+            deletePlaylist(binding.playlistName.text.toString())
+        }
+        binding.editPlaylist.setOnClickListener {
+            findNavController().navigate(
+                R.id.action_playlistViewFragment_to_playlistCreationFragment,
+                Bundle().apply {
+                    putBoolean(PlaylistCreationFragment.FROM_NAVCONTROLLER_KEY, true)
+                    putInt(PlaylistCreationFragment.PLAYLIST_ID_KEY, playlistId)
+                })
+        }
+        findNavController().currentBackStackEntry?.savedStateHandle?.getLiveData<Any>(
+            PlaylistCreationFragment.RESULT
+        )?.observe(viewLifecycleOwner) { _ ->
+            synchronized(this) {
+                viewModel.loadPlaylist(playlistId)
+            }
+        }
+
+            viewModel.observeAllTracks().observe(viewLifecycleOwner) { tracks ->
+            if (!tracks.isNullOrEmpty()) {
+                binding.listItems.visibility = View.VISIBLE
+                binding.nothing.visibility = View.GONE
+                binding.recyclerView.visibility = View.VISIBLE
+                val durationInMinutes = (tracks.sumOf { it.trackTime } / 60000).toInt()
+                val formattedDuration = String.format("%d", durationInMinutes)
+                val minuteWord = getMinutePluralForm(durationInMinutes)
+                val trackWord = getPluralForm(tracks.size).format(tracks.size)
+                binding.recyclerView.isGone = false
+                binding.emptyPl.isVisible = false
+
+                binding.playlistInfo.text = getString(
+                        R.string.pl_info,
+                        "$formattedDuration $minuteWord",
+                        trackWord
+                    )
+                adapter.updateTracks(tracks)
+                binding.share.setOnClickListener {
+                    sharePlaylist()
```

Comment:
И здесь тоже нужно скрывать
```suggestion
                    sharePlaylist()
                    bottomMenuBehavior.state = BottomSheetBehavior.STATE_HIDDEN
```

Reply:
Так тут нижнее меню не всплывает?